app/helpers/application_helper.rb

```diff
@@ -1,4 +1,14 @@
 # frozen_string_literal: true
 
 module ApplicationHelper
+  def make_this_a_list(options: {})
+    options[:document] # the original document
```

Comment:
What are these three lines doing?

Reply:
Hah, I copied from https://github.com/projectblacklight/blacklight/wiki/Configuration---Results-View#helper-method. I assumed they were doing some Ruby/Rails/Blacklight magic but maybe they were just demonstrating what was available in options ...

Reply:
Yes, testing now.

Reply:
@jcoyne Removed.